pkgs/by-name/py/pyamlboot/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pyamlboot";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "superna9999";
+    repo = "pyamlboot";
+    rev = "refs/tags/${version}";
+    hash = "sha256-vpWq8+0ZoTkfVyx+2BbXdULFwo/Ug4U1gWArXDfnzyk=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    pyusb
+    wheel
+    setuptools
+  ];
+
+  nativeInputChecks = [
```

Comment:
```suggestion
  nativeCheckInputs = [
```

Reply:
What's the failure about?

Reply:
The pytestCheckHook fails, cause there aren't any tests